addons/account/static/src/components/product_label_section_and_note_field/product_label_section_and_note_field.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+
+    <t t-name="account.ProductLabelSectionAndNoteField">
+        <t t-if="props.readonly">
+            <t t-if="!props.canOpen">
+                <div class="o_product_label_section_and_note_field_cell">
+                    <t t-if="hasProduct and displayName">
+                        <span t-esc="displayName"/>
+                        <button
+                            class="btn fa fa-bars product_label_button"
+                            data-tooltip="Click to see and edit description"
+                            t-on-click="(ev) => this.extendLines()"/>
+                        <t t-if="extended and extraLines">
+                            <br/>
+                            <input
+                                class="o_input o_product_label_section_and_note_field_label_input"
+                                t-att-value="extraLines"
```

Comment:
Looks a bit sus to me
`extraLines` is returning an array, is it working? :thinking: 
The parent component is using the `extraLines` in `t-foreach` not for a `t-att-value`.

Reply:
If multiple lines are to be shown, my current method shows lines next to each other and separated by commas. I guess this is not what we want but i'm not sure if we should have one line by line of label as it could rapidly make it less readable than it was at first. I'm gonna ask the PO.

Reply:
Answer from the PO: Keep user's line breaks